css/style.css

```diff
@@ -1,11 +1,365 @@
-@import url("https://fonts.googleapis.com/css2?family=Metal+Mania&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Manrope:wght@200;800&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
 
 body {
-  background-color: black;
+  font-family: "Manrope", "Poppins", sans-serif;
+  background: #e5e5e5;
+}
+
+.p-rel {
+  position: relative;
+}
+
+.p-absl {
+  position: absolute;
+}
+
+.p-fixed {
+  position: fixed;
+}
+
+.font-size-40 {
+  font-size: 40px;
+}
+
+.font-size-28 {
+  font-size: 28px;
+}
+
+.font-size-24 {
+  font-size: 24px;
+}
+
+.font-size-18 {
+  font-size: 18px;
+}
+
+.font-size-16 {
+  font-size: 16px;
+}
+
+.font-size-12 {
+  font-size: 12px;
+}
+
+.font-size-8 {
+  font-size: 8px;
+}
```

Comment:
Noted 👍🏽  I think the project documentation says otherwise
![Screenshot (33)_LI](https://user-images.githubusercontent.com/44624138/124952495-34808780-e00c-11eb-8822-b2af79c9ed22.jpg)

Reply:
That's why it's an optional suggestion. On the Figma design, they will use ```px``` unit but it si upon the developer to make the font-sizing responsive.